var/spack/repos/builtin/packages/reportinglib/package.py

```diff
@@ -33,7 +33,9 @@ class Reportinglib(CMakePackage):
     homepage = "https://bbpcode.epfl.ch/code/a/sim/reportinglib/bbp"
     url      = "ssh://bbpcode.epfl.ch/sim/reportinglib/bbp"
 
-    version('develop', git=url, tag='2.5.0', preferred=True)
+    version('develop', git=url, branch='master')
+    version('2.5.0', git=url, tag='2.5.0')
+    version('2.5.1', git=url, tag='2.5.1', preferred=True)
```

Comment:
Sort order? We normally have them in descending order, i.e., these two versions should be swapped?

Reply:
done